.travis.yml

```diff
@@ -1,3 +1,12 @@
+before_script:
+  - npm install -g grunt-cli
+
+sudo: false
+
+after_script:
+  - grunt coveralls
+
 language: node_js
 node_js:
-  - "0.10"
\ No newline at end of file
+  - "4.0"
```

Comment:
Good catch on the engine...I have travis testing both right now so Ill change the engine to include the stable version also.